core-1/src/main/java/scanning/parkinglot/ProjectConfig.java

```diff
@@ -0,0 +1,10 @@
+package scanning.parkinglot;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan(basePackages = ("scanning/parkinglot"))
```

Comment:
remove configuration class, application context should be loaded with scanning feature